### ISSUE TITLE
Check if system image belongs to the account

### ISSUE THREAD

**Issue by smirnp** [hobbit-project/platform#313]
Hi! 

Currently there is no checking, if a challenge participant registers a system mapped image of some another account.

We just figured out, that some participants forgot to their change imageName in system.ttl (and used my dummy system uploaded under my account). This critical for security.

**Comment by MichaelRoeder** [comment]
This is not a bug. Its a feature :wink: 

Users can register all systems they can "see". So your participants simply have to make sure that their git projects containing the `system.ttl` are `privat`. Otherwise they are visible for other, registered users.

**Comment by smirnp** [comment]
The problem is not in registration, but in execution. 

I have a system, which is a private and nobody can register that for a challenge.

But anybody can see find the url of it (e.g. in Status tab) and put it in his system.ttl file and my docker image will be executed as somebody's else system.

**Comment by MichaelRoeder** [comment]
No, this is not easily possible since the platform checks whether a URI is used by more than 1 system, although their might be some ways to get around this.

**Comment by MichaelRoeder** [comment]
So there is a problem in the "direction" in which this protection is working. The idea was that it blocks the other user from using its system adapter instead of stopping you from using it :thinking:

**Comment by smirnp** [comment]
The warning is displayed not only to owners of a system. See: https://github.com/hobbit-project/DEBS-GC-2018/issues/47 
I can confirm this from my another account
